packages/core/src/parse/tokens-studio.ts

```diff
@@ -365,10 +369,65 @@ export function convertTokensStudioFormat(rawTokens: Record<string, unknown>): {
               tokenPath,
             );
           } else {
+            let value: string | undefined = v.value;
```

Comment:
This is pretty hacky, but fixes an existing bug where `rgba({colors.black}, 0.2)` would get left as-is.

Does a _little_ work to fix common scenarios, but is by no means exhaustive and there are still plenty of other broken scenarios I’m sure.